js/lib/beautify-html.js

```diff
@@ -119,7 +121,9 @@
             : 0;
         indent_handlebars = (options.indent_handlebars === undefined) ? false : options.indent_handlebars;
         end_with_newline = (options.end_with_newline === undefined) ? false : options.end_with_newline;
-
+        extra_liners = Array.isArray(options.extra_liners) ?
+            options.extra_liners : (typeof options.extra_liners === 'string') ?
```

Comment:
`options.extra_liners.concat() : ` - You want a copy of the array, not a reference.  Otherwise, someone could change the array after it was passed in.